annif/cli.py

```diff
@@ -551,5 +551,21 @@ def run_hyperopt(project_id, paths, docs_limit, trials, jobs, metric, results_fi
     click.echo("---")
 
 
+@cli.command("completion")
+@click.option("--bash", "shell", flag_value="bash")
+@click.option("--zsh", "shell", flag_value="zsh")
+@click.option("--fish", "shell", flag_value="fish")
+def completion(shell):
+    """Generate autocompletion script for the given shell."""
+
+    if shell is None:
+        click.echo("Shell not given, try --bash, --zsh or --fish")
+        return
+
+    script = os.popen(f"_ANNIF_COMPLETE={shell}_source annif").read()
+    click.echo("# Generated by Annif")
```

Comment:
How about adding the Annif version number to the comment? Later versions may have more commands etc.